### PR TITLE
Java 15

### PR DESCRIPTION
**Pull request by Sam-Kramer**

### CODE REVIEW

**Review by leonz — APPROVED** [review]

File: bin/cassandra.in.sh [hunk 1]
```diff
@@ -41,15 +41,15 @@ cassandra_bin="$cassandra_bin:$CASSANDRA_HOME/build/classes/thrift"
...
-    JAVA_HOME="${JAVA_11_HOME}"
+if [ -n "${JAVA_15_HOME}" ]; then
+    JAVA_HOME="${JAVA_15_HOME}"
 fi
 
-# verify that JAVA_HOME points to java 11
+# verify that JAVA_HOME points to java 15
 java_ver_output=`"$JAVA_HOME/bin/java" -version 2>&1 | grep '[openjdk|java] version' | awk -F'"' 'NR==1 {print $2}'`
 jvm_version=${java_ver_output%_*}
-if [ "$jvm_version" \< "11.0" ] ; then
```
Comment (leonz) [inline bin/cassandra.in.sh:51]: any chance we want to support java 11 + 15? is there any chance that some deployments still don't use java 15?

Comment (leonz) [reply bin/cassandra.in.sh:51]: especially given there are no actual code changes

Comment (Sam-Kramer) [reply bin/cassandra.in.sh:51]: Happy to change to 11, although I'd imagine every deployment has java 15

Comment (leonz) [reply bin/cassandra.in.sh:51]: 👍

**Pull request merged**